homeassistant/components/music_assistant/media_player.py

```diff
@@ -219,7 +221,9 @@ async def async_on_update(self) -> None:
                     )
                 )
             ]
-        self._attr_group_members = group_members_entity_ids
+        # NOTE: we sort the group_members for now,
+        # until the MA API returns them sorted (group_childs is now a set)
+        self._attr_group_members = sorted(group_members_entity_ids)
```

Comment:
Without this, the snapshot keeps failing every once in a while due to the group_members list having a different order. I'll patch MA to no longer use a set for this in the future.

Reply:
ma client is patched so this can be removed when we bump the mass client lib